### DISCUSSION TITLE
Change the originals folder

### DISCUSSION THREAD

**Discussion by londonpete1958** [photoprism/photoprism#4066] [Q&A]
I set the original folder in the yaml to
"/share/Multimedia/photos:/photoprism/originals" which has got hundreds of photos in it. However, when I run the index option, it says 0 files.

How can I check exactly where the originals is pointing at and change it if I need to.

Many thanks
Pete

**Comment by lastzero** [comment]
Open a terminal via "docker compose exec photoprism bash" to manually check what files and directories have been mounted. Also see the troubleshooting checklists we provide for this purpose:
- https://docs.photoprism.app/getting-started/troubleshooting/#missing-pictures

**Comment by londonpete1958** [comment]
Thanks lastzero,

What folder should I be in to run the docker command? Wherever I try it I get "no configuration file provided: not found"

Regards
Pete

**Reply by ai30328348** [reply]
In the Potoprism folder containing the docker-compose.yml
